chart/templates/ingress.yaml

```diff
@@ -61,7 +61,7 @@ spec:
               servicePort: {{ $streamingPort }}
               {{- end }}
             {{- if or ($.Capabilities.APIVersions.Has "networking.k8s.io/v1/Ingress") (not ($.Capabilities.APIVersions.Has "networking.k8s.io/v1beta1/Ingress")) }}
-            pathType: ImplementationSpecific
```

Comment:
Streaming breaks on nginx ingress with this change 😢

Reply:
I actually didn't test this properly with istio. ImplementationSpecific and Exact don't work with istio for some reason. Prefix does though so maybe you could check with Prefix again?

Reply:
Potentially also just a trailing slash is missing

Reply:
It seems like it works with a trailing slash and exact matching. So maybe you could test that instead please and decide whether that would be an acceptable change?

Reply:
trailing slash and exact matching works for me on nginx 👍

Reply:
I wonder if this is nginx ingress doing the unexpected.
From https://kubernetes.io/docs/concepts/services-networking/ingress/#examples it really sounds like the current setup should work. Although the same documentation suggests that adding a trailing slash shouldn't break anything so :woman_shrugging:

Reply:
I don't fully follow you. The page says
> ImplementationSpecific: With this path type, matching is up to the IngressClass. Implementations can treat this as a separate pathType or treat it identically to Prefix or Exact path types.

and

![image](https://user-images.githubusercontent.com/13287984/200135316-05fac4cf-be36-4f7f-9b23-a371bd0de71f.png)

so it seems like we could either use Prefix matching without a trailing slash (didn't test nginx but should work) or exact matching with a trailing slash

Reply:
My eyes are too tired to read tables, you are correct. I've tested it with trailing slash on nginx and that works